public/locations-list-map.js

```diff
@@ -602,12 +597,8 @@ function loadMapScript(searchParams, data, filters) {
   const apiKey = 'AIzaSyDSz0lnzPJIFeWM7SpSARHmV-snwrAXd2s';
   let scriptSrc = `//maps.googleapis.com/maps/api/js?libraries=geometry,places&callback=initMap&key=${ apiKey }`;
 
-  const currentLocale = searchParams.get('locale') || 'en-US';
-  const [language, region] = currentLocale.split('-');
-
-  if (language) {
-    scriptSrc += `&language=${ language }&region=${ region }`;
-  }
+  const languageRegion = getMapsLanguageRegion();
+  scriptSrc += `&language=${languageRegion.language}&region=${languageRegion.region}`;
```

Comment:
Now that language/region is always present, you can move this up and build scriptSrc with one template string, instead of concatenating.